### ISSUE TITLE
Bug: cancel header edit doesn’t work

### ISSUE THREAD

**Issue by blueturtleai** [Dimillian/IceCubesApp#1956]
## Environment:
iOS 17.2 icecubes

## Description
Cancelling the edit of the profile header saves the uploaded image instead of discarding it.

**Comment by mdhsieh** [comment]
Yeah I can see the same thing on iOS 17.0. I'll go ahead and work on this.
